homeassistant/components/overkiz/__init__.py

```diff
@@ -159,6 +176,43 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    async def handle_execute_command(service: ServiceCall) -> ServiceResponse:
+        """Handle execute command service."""
+        entity_registry = er.async_get(hass)
+
+        for entity_id in service.data[ATTR_ENTITY_ID]:
+            registry_entity = entity_registry.async_get(entity_id)
+            entity: OverkizEntity = [
+                e
+                for e in hass.data["entity_components"][registry_entity.domain].entities
+                if e.unique_id == registry_entity.unique_id
+            ][0]
```

Comment:
This is very very ugly, but I couldn't find a helper to retrieve the actual entity from the RegistryEntity. I need to retrieve my coordinator from my entity, otherwise for users with 2 Config Entries, this service call might run on the wrong config entry...